src/v2/Apps/Collect/Utils/collectionToArtistSeriesSlugMap.tsx

```diff
@@ -1,122 +1,1201 @@
+  "ben-eine-love": "ben-eine-love",
+  "ben-eine-revolution": "ben-eine-revolution",
+  "ben-eine-riot": "ben-eine-riot",
+  "ben-eine-scary": "ben-eine-scary",
+  "ben-eine-soho": "ben-eine-soho",
+  "ben-frost-krispy-kreme": "ben-frost-krispy-kreme",
+  "ben-frost-mcdonalds": "ben-frost-mcdonalds",
+  "bernd-and-hilla-becher-blast-furnaces":
+    "bernd-and-hilla-becher-blast-furnaces",
+  "bernd-and-hilla-becher-gas-tanks": "bernd-and-hilla-becher-gas-tanks",
+  "bernd-and-hilla-becher-grain-elevators":
+    "bernd-and-hilla-becher-grain-elevators",
+  "bernd-and-hilla-becher-industrial-facades":
+    "bernd-and-hilla-becher-industrial-facades",
+  "bernd-and-hilla-becher-water-towers": "bernd-and-hilla-becher-water-towers",
+  "bernd-and-hilla-becher-winding-towers":
+    "bernd-and-hilla-becher-winding-towers",
+  "betty-tompkins-apologia": "betty-tompkins-apologia",
+  "betty-woodman-vases": "betty-woodman-vases",
+  "blek-le-rat-last-tango": "blek-le-rat-last-tango",
+  "blek-le-rat-mona-lisa": "blek-le-rat-mona-lisa",
+  "blek-le-rat-the-warrior": "blek-le-rat-the-warrior",
+  "brice-marden-cold-mountain-series": "brice-marden-cold-mountain-series",
+  "brice-marden-red-yellow-blue": "brice-marden-red-yellow-blue",
+  "bridget-riley-blaze": "bridget-riley-blaze",
+  "bridget-riley-dots": "bridget-riley-dots",
+  "bridget-riley-fragments": "bridget-riley-fragments",
+  "bridget-riley-stripes": "bridget-riley-stripes",
+  "bruce-nauman-clowns": "bruce-nauman-clowns",
+  "bruce-nauman-fingers-and-holes": "bruce-nauman-fingers-and-holes",
+  "bruce-nauman-holograms": "bruce-nauman-holograms",
+  "candida-hofer-libraries": "candida-hofer-libraries",
+  "carl-andre-poems": "carl-andre-poems",
+  "carlos-cruz-diez-couleur-additive": "carlos-cruz-diez-couleur-additive",
+  "carlos-cruz-diez-physichromie": "carlos-cruz-diez-physichromie",
+  "carlos-cruz-diez-serie-cardiaca": "carlos-cruz-diez-serie-cardiaca",
+  "carlos-cruz-diez-serie-semana": "carlos-cruz-diez-serie-semana",
+  "carrie-mae-weems-africa-series": "carrie-mae-weems-africa-series",
+  "carrie-mae-weems-kitchen-table-series":
+    "carrie-mae-weems-kitchen-table-series",
+  "carrie-mae-weems-louisiana-project": "carrie-mae-weems-louisiana-project",
+  "carrie-mae-weems-not-manets-type": "carrie-mae-weems-not-manets-type",
+  "catherine-opie-domestic": "catherine-opie-domestic",
+  "catherine-opie-football": "catherine-opie-football",
+  "catherine-opie-modernist": "catherine-opie-the-modernist",
+  "chris-burden-bridges": "chris-burden-bridges",
+  "chris-ofili-afro-lunar-lovers": "chris-ofili-afro-lunar-lovers",
+  "christo-and-jeanne-claude-the-gates": "christo-jeanne-claude-the-gates",
+  "christo-floating-piers": "christo-jeanne-claude-floating-piers",
+  "christo-wrapped-reichstag": "christo-jeanne-claude-wrapped-reichstag",
+  "christo-wrapped-statues": "christo-jeanne-claude-wrapped-statues",
+  "christo-wrapped-volkswagen": "christo-jeanne-claude-wrapped-volkswagen",
```

Comment:
I suspect the diff won't be very amenable to reviewing 🤷 but this is based on the output of the original migration + some hand touchups for multiple-artist series.

I've spot checked a dozen or so of these locally, both the ones that were 1:1 as well as the ones that changed in the migration, like these^